### ISSUE TITLE
Quick Start: Blue Text in Notices

### ISSUE THREAD

**Issue by diegoreymendez** [wordpress-mobile/WordPress-Android#9853]
## Description:

In iOS, when going through the guided steps to customize the site and to grow the audience, we show a snackbar that highlights the item you have to tap in the dashboard.

In Android, the snackbar has no color to highlight what you need to tap.

## Images:

![IMG_9E2B15E903E0-1](https://user-images.githubusercontent.com/1836005/57664172-81ec4000-75cd-11e9-9b36-a074b6d40f86.png)

![Screenshot_20190513-213714](https://user-images.githubusercontent.com/1836005/57664158-700a9d00-75cd-11e9-964b-288de3f41228.png)

## Related:

https://github.com/wordpress-mobile/WordPress-iOS/issues/11688

**Comment by diegoreymendez** [comment]
Can I have your input on this one @SylvesterWilmott ?

**Comment by SylvesterWilmott** [comment]
This seems directly related to #11688 

It seems at some point the Android version lost it's formatting because it was the same as iOS previously.

I think we can use white. bold text to highlight the action within the snackbars while keeping the rest of the text grey:

<img width="375" alt="Screenshot 2019-05-15 at 09 45 01" src="https://user-images.githubusercontent.com/36532468/57761839-3e412580-76f6-11e9-86ee-abe1ede96008.png">

**Comment by malinajirka** [comment]
I've just tested this on `develop` and the Android app uses `hot_pink_300`. I assume @diegoreymendez took the screenshot on a device with Android api < 24 on which the feature doesn't work. (I've created a [separate issue](https://github.com/wordpress-mobile/WordPress-Android/issues/9897))

I've created a PR which replaces the "hot_pink" with "white" and adds "Bold" style to the highlighted item.

I've also renamed this issue and I've removed the "Site Creation" label as it's not related to site creation. I've kept it in the Site Creation project so we can track the progress.
